mail_tracking/tests/test_mail_tracking.py

```diff
@@ -108,10 +113,6 @@ def test_message_post(self):
         self.assertEqual(tracking_email.state, "sent")
         # message_dict read by web interface
         message_dict = message.message_format()[0]
-        self.assertTrue(len(message_dict["partner_ids"]) > 0)
-        # First partner is recipient
-        partner_id = message_dict["partner_ids"][0]
-        self.assertEqual(partner_id, self.recipient.id)
```

Comment:
Why removing this assertions?

Reply:
This is because, in `v16`, `message_format` no longer returns `partner_ids`, see v16's `_get_message_format_fields`:

https://github.com/odoo/odoo/blob/6bb16bf8818042b05ff8d40f852fa8e7d9eba92f/addons/mail/models/mail_message.py#L978

vs v15's `_get_message_format_fields`:

https://github.com/odoo/odoo/blob/c59d6e109021c72d32884793d48ec60fecb95168/addons/mail/models/mail_message.py#L979

So the assertion raises an error unrelated to `mail_tracking` :)